src/server/game/Quests/QuestDef.h

```diff
@@ -257,7 +257,7 @@ class TC_GAME_API Quest
         std::string const& GetRequestItemsText() const { return _requestItemsText; }
         std::string const& GetAreaDescription() const { return _areaDescription; }
         std::string const& GetCompletedText() const { return _completedText; }
-        int32  GetRewOrReqMoney() const;
+        int32  GetRewOrReqMoney(const Player* player = nullptr) const;
```

Comment:
`Player const*`

Reply:
Fixed, thanks!

(I'll never get used to that codestyle rule, I've been reading and using const type* for years :P)